helm/hpcc/values.schema.json

```diff
@@ -508,6 +511,31 @@
       "required": [ "name", "prefix", "category" ],
       "additionalProperties": false
     },
+    "remoteStorage": {
```

Comment:
This doesn't match what I expect.  For instance if the remote system uses multiple storage planes, how does the caller configure, or know which storage plane it is mapped to?

I was expecting something like
remote:
- name: xxx
  service: ...
  planes:
    remote-name: local-name

Or a similar structure/list that allows multiple planes to be mapped for a remote cluster.

Reply:
you're right, I was going to say in current scheme you need to define a 'remote' mapping for each remote plane, even if in same environment.. 
But you're right, it doesn't cater for the meta info from a single remote environment pointing to different planes, and it needs a mapping remote-plane: local-plane. At the moment it's assuming all map to the same one.
Will rework it handle.

Reply:
@ghalliday - pushed changes.